soc/xtensa/nxp_adsp/imx8/include/soc/memory.h

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright (c) 2021 NXP Semiconductors
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef __INC_MEMORY_H
+#define __INC_MEMORY_H
+
+#define PLATFORM_CORE_COUNT 1
+
+/** Id of master DSP core */
+#define PLATFORM_PRIMARY_CORE_ID	0
+
+#define SRAM_RESERVE_HEADER_SPACE	0x00010000
+
+#define SRAM_BASE	0x596E8400
```

Comment:
SRAM_BASE is only 1k-aligned?  Is that correct?  Or is this a way of getting code out of the way of something special at the bottom of memory?  If so, calling it "base" is maybe confusing.

Reply:
This is an unfortunate copy-paste.
The address is the iMX IRAM address, but is never used.

Reply:
I've removed it.